### PR TITLE
Add user profile

### PR DESCRIPTION
**Pull request by jingjingwang**

### CODE REVIEW

**Comment by jingjingwang** [comment]
Also solves #1.

File: civictechprojects/templates/project_signup.html [hunk 1]
```diff
@@ -69,32 +69,32 @@ <h2 class="form-group subheader">DETAILS</h2>
 
             <div class="form-group">
                 <label for="brief_description">Describe This Project</label>
-                <textarea class="form-control" id="brief_description" placeholder="This will appear as project introduction" rows=3></textarea>
+                <textarea class="form-control" name="brief_description" placeholder="This will appear as project introduction" rows=3></textarea>
```
Comment (marlonkeating) [inline civictechprojects/templates/project_signup.html]: This wasn't added by your change, but I noticed it during testing.  Name should be "project_description" in order for it to be saved by the view.

**Comment by jingjingwang** [comment]
Added a few fixes. One thing that I'm not entirely sure is that, should we use an email address as a username. Few places would need to be revisited if one day we changed the design.

**Comment by marlonkeating** [comment]
User profile design has changed, so closing this review.

**Pull request closed without merging**